runtime/interpreter/value.go

```diff
@@ -9508,3 +9524,22 @@ func NewPublicAccountKeysValue(getFunction FunctionValue) *CompositeValue {
 		fields:              fields,
 	}
 }
+
+func checkContainerMutation(
```

Comment:
Minor: Maybe make the interpreter the receiver, i.e. `func (inter *Interpreter) checkContainerMutation(...`

Reply:
Added with 4af18e7